quiz/backend/config/settings_logging.py

```diff
@@ -0,0 +1,51 @@
+import os
+from pathlib import Path
+
+BASE_DIR = Path(__file__).resolve().parent.parent
```

Comment:
zaimportuj to z settings.py, bo inaczej jak zmieni się struktura katalogów, to trzeba będzie poprawiać w 2 miejscach

Reply:
Zmienione, ale męczył mnie circular error, więc w settings_logging pojawiła się fn do zwracania configa, można by było zrobić BASE_DIR jako zmienną środowiskową, ale byśmy musieli mieć brzydki import w settings.py